main/src/cgeo/geocaching/connector/lc/LCApi.java

```diff
@@ -166,7 +166,9 @@ private static Geocache parseCache(final JsonNode response) {
             cache.setDifficulty((float) 1);
             cache.setTerrain((float) 1);
             cache.setSize(CacheSize.getById("virtual"));
-            cache.setFound(false);
+            if (DataStore.isThere("LC" + uuid, "", false)) {
+                cache.setFound(false);
+            }
             DataStore.saveCache(cache, EnumSet.of(SaveFlag.CACHE));
```

Comment:
not sure why the condition shouldn't read `if (!DataStore ....)`. I thought `isThere` returns `true`if the cache already is in datastore in which case we cont want to do the `setFound(false)`, however, the way it is coded now just "works" as intended.

Reply:
I guess this is a reason of how `Geocache` objects get merged in memory vs. DB. Have you checked if leaving `cache.setFound()` out completely would work as well (which would be the cleaner way)?

Reply:
Actually I was preparing for the unlikely case that we might run the API in active mode in which case we get the found state from the API, the code reads now:

```
final Boolean isComplete = response.get("IsComplete").asBoolean();
...
cache.setFound(isComplete);
```

Reply:
pushed a new commit addressing your findings

Reply:
_If_ we want to / can support active mode, the found state should always be applied and not only if cache is stored. But as long as we are in passive mode, `found` should be `null` (means no value set, by which the previous found state will always be automatically taken over). 

If you want to, you can add a small comment instead 
(something like: `// cache.setFound(response.get("IsComplete").asBoolean()); as soon as we're using active mode`)

Reply:
Done